app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.ProfileChooserViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.core.widget.NestedScrollView
```

Comment:
Do we really want part of the view to remain fixed, or should the whole screen scroll?

Reply:
@BenHenning Except the bottom portion we want the remaining portion to scroll PTAL https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/1fc109c3-31f3-4743-bd3b-76be39c0049a/PC-SP-Profile-Chooser-PIN-Available-

Reply:
Hmm I see. If this works okay, let's keep it as-is. I'm always a bit hesitant to used NestedScrollView with nested RecyclerView since the scrolling can get a bit wonky, but if this layout is simple enough then let's not change anything. One alternative approach would have been to just put everything except the controls in the recycler view, but let's not make that change if it isn't necessary.

Thanks for clarifying.

Reply:
@BenHenning Okay. As if now i'am not changing. If you still want me to change after reviewing the app changes, let me know.

Reply:
Resolving since we're keeping this solution. Thanks.